libexec/rtld-elf/rtld.1

```diff
@@ -26,7 +26,7 @@
 .\" (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 .\" THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .\"
-.Dd July 24, 2025
+.Dd July 24, 2024
```

Comment:
This file was edited on 2024-07-14 and 2024-07-25 (going by author date) in 860c4d94ac46cee35a678cf3c9cdbd437dfed75e and bc25bc6850e512c4c45effdff6178d20b5eca558, which set the date to "July 24, 2025".  I'm willing to bet that the date 24 vs. 25 is a timezone issue.

And it's pretty clear that the year 2025 was a typo for 2024.